### PR TITLE
Add functionality for book creating and deleting

### PR DESCRIPTION
**Pull request by bettercallazamat**
In this PR, I have connected the CREATE_BOOK and REMOVE_BOOK actions with the React component and made the page dynamic.

### CODE REVIEW

File: src/containers/BooksForm.js [hunk 1]
```diff
@@ -7,10 +11,33 @@ const BooksForm = () => {
...
+    }
+    if (e.target.name === 'category') {
+      category = e.target.value;
+      setBook((state) => ({ ...state, category }));
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(createAction({ ...book, id: Math.floor(Math.random() * 100) }));
```
Comment (rindrajosia) [inline src/containers/BooksForm.js:33]: - [x] [REQUIRED] you should `reset the STATE `once the book is created.  [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/HandleSubmit.png)

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] I got errors related to PropTypes when I open your APP. Keep in mind that PropTypes are a mechanism to ensure that components use the correct data type and pass the right data, and that components use the right type of props, and that receiving components receive the right type of props.



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by bettercallazamat** [comment]
> Hi,
> 
> Good job so far!
> There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! 💪
> 
> ## Required Changes ♻️
> * [ ]  I got errors related to PropTypes when I open your APP. Keep in mind that PropTypes are a mechanism to ensure that components use the correct data type and pass the right data, and that components use the right type of props, and that receiving components receive the right type of props.
>   
> 
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Could you give a hint on how can I check prop types correctly? Reason why I am asking, because when I use PropTypes.shape or book.object I am getting eslint errors.

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**